index.js

```diff
@@ -643,8 +643,30 @@ class ServerlessAppsyncPlugin {
           RelationalDatabaseSourceType: ds.config.relationalDatabaseSourceType || 'RDS_HTTP_ENDPOINT',
         };
       } else if (ds.type === 'HTTP') {
+        const authorizationConfig = {
+          ...(ds.config.authorizationConfig && {
+            AuthorizationConfig: {
+              ...(ds.config.authorizationConfig.authorizationType && {
+                AuthorizationType: ds.config.authorizationConfig.authorizationType,
+              }),
+              ...(ds.config.authorizationConfig.awsIamConfig && {
+                AwsIamConfig: {
+                  ...(ds.config.authorizationConfig.awsIamConfig.signingRegion && {
+                    SigningRegion: ds.config.authorizationConfig.awsIamConfig.signingRegion,
```

Comment:
@yai333 Can we default the signing region to the stack's region? with fallback to account region? 
a bit like here: https://github.com/sid88in/serverless-appsync-plugin/blob/master/index.js#L287

Reply:
updated as per request, please check